server/server.js

```diff
@@ -122,6 +130,23 @@ var tfamatFile = {
 app.post('/genotet', function(req, res) {
   var type = req.body.type;
   console.log('POST', type);
+
+  switch(type) {
+    // upload
+    case 'upload':
+      var fileType = req.body.type;
```

Comment:
Looks like this conflicts with the type above? Maybe name it fileType for the moment.